libbpf-rs/src/program.rs

```diff
@@ -864,6 +878,96 @@ impl<'obj> ProgramMut<'obj> {
         Ok(link)
     }
 
+    fn check_kprobe_multi_args<T: AsRef<str>>(symbols: &[T], cookies: &[u64]) -> Result<usize> {
+        if symbols.is_empty() {
+            return Err(Error::with_invalid_data("Symbols list cannot be empty"));
```

Comment:
Let's introduce and use `Error::with_invalid_input` here and below. These are effectively user input issues.